Tests/Unit/System/Configuration/ExtensionConfigurationTest.php

```diff
@@ -74,8 +74,8 @@ public function testIsGetIsUseConfigurationMonitorTablesConfigured()
             ['useConfigurationMonitorTables' => 'pages, tt_content']
         );
         $expectedResult = [
-            'pages' => true,
-            'tt_content' => true
+            'pages' => 0,
```

Comment:
Maybe I just misunderstood the intention of this https://github.com/TYPO3-Solr/ext-solr/pull/1123#discussion_r105605392? But you can discard the PR if you mean it's not relevant.